src/app/pages/StoryPage/__snapshots__/index.test.jsx.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`Story Page should not show the pop-out timestamp when allowDateStamp is false 1`] = `
 <DocumentFragment>
-  .c54 {
+  .c59 {
```

Comment:
Masses of changes in this file - hard to tell from the diff, is it expected?

Reply:
I don't know whyyyyy 😢 I have tried deleting the snapshot & regenerating it, doing an `npm install` - I can't tell why these classes have changed. If I don't update the snapshots, tests fail.

Reply:
Aye could be nothing to worry about - is the entire file just rearranging the existing snapshot/renaming existing classes, or is it actually adding/removing elements?